.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: All the CI
+
+on: [push, pull_request]
```

Comment:
Thinking of changing this to only pushing to master, and PRs - thoughts?

Reply:
Seems OK, but I don't see the need we have to restrict

Reply:
I had a reason but I don't anymore 🙃

Reply:
You can drop the `pull_request` trigger or it will trigger twice.

Reply:
It will only trigger for branches from this repo that are PR'd, correct? E.g. if you create a PR from your fork, CI would never run. I think that might have been why I only wanted CI to run on master + PRs

Reply:
You are right ! Let's re-add the master specification for push event.